vpn-proxy/app/models.py

```diff
@@ -20,30 +21,47 @@
 
 
 def chose_server_ip(network=VPN_ADDRESSES):
-    """Find an available server IP in the given network (CIDR notation)"""
-    network = netaddr.IPNetwork(network)
-    if network.version != 4 or not network.is_private():
-        raise Exception("Only private IPv4 networks are supported.")
-    first, last = network.first, network.last
-    if first % 2:
-        first += 1
-    for i in range(20):
-        addr = str(netaddr.IPAddress(random.randrange(first, last, 2)))
-        print addr
-        try:
-            Tunnel.objects.get(server=addr)
-        except Tunnel.DoesNotExist:
-            return addr
+	"""Find an available server IP in the given network (CIDR notation)"""
+	network = netaddr.IPNetwork(network)
+	if network.version != 4 or not network.is_private():
+		raise Exception("Only private IPv4 networks are supported.")
+	first, last = network.first, network.last
+	if first % 2:
+		first += 1
+	for i in range(20):
+		addr = str(netaddr.IPAddress(random.randrange(first, last, 2)))
+		print addr
+		try:
+			Tunnel.objects.get(server=addr)
+		except Tunnel.DoesNotExist:
+			return addr
```

Comment:
Why did you mess with my indentation. You should be using 4 spaces for indentation, not tabs. https://www.python.org/dev/peps/pep-0008/#indentation